hazelcast-client/src/main/java/com/hazelcast/client/connection/ClientConnectionManager.java

```diff
@@ -26,25 +26,32 @@
 import java.util.Collection;
 
 /**
- * Responsible for managing {@link com.hazelcast.client.connection.nio.ClientConnection} objects.
+ * Responsible for managing {@link com.hazelcast.client.connection.nio.ClientConnection} instances.
  */
 public interface ClientConnectionManager extends ConnectionListenable {
 
     /**
-     * Check if client connection manager is alive.
+     * Check this ClientConnectionManager is alive.
      * ClientConnectionManager is not alive only when client is closing.
      *
      * @return true if alive, false otherwise.
```

Comment:
thread-safe

Reply:
What does that mean? That I should add thread safe to each of these methods?